README.md

```diff
@@ -2,6 +2,9 @@
 ilib-webos-loctool-qml is a plugin for the loctool allows it to read and localize qml files. This plugins is optimized for webOS platform.
 
 ## Release Notes
+v1.3.5
+* Fix to be added js file extension to path list for parsing
```

Comment:
"Add 'js' to the list of file extensions that this plugin handles"

Reply:
Thanks! :)